servicebroker/config/catalog.schema.json

```diff
@@ -201,13 +201,8 @@
                   "description": "The interval between two successive scaling activity"
                 },
                 "metric_type": {
-                  "enum": [
-                    "memoryused",
-                    "memoryutil",
-                    "responsetime",
-                    "throughput"
-                  ],
-                  "type": "string"
+                  "type": "string",
+                  "pattern": "^[a-zA-Z0-9_]+$"
```

Comment:
the metric type can start with a number? such as "999metric".

Reply:
@pradyutsarma @paltanmoy @boyang9527 @cdlliuy 
should custom metric name be allowed to start with number ?

Reply:
I think that is fine